### ISSUE TITLE
When using sugarss and a URL statement compilation fails

### ISSUE THREAD

**Issue by davidpelaez** [postcss/postcss-loader#64]
I just tried to compile this:

```
body
 background-color: red
 background-image: url('lobo.png')

.butoncito
 color: red
```

In Webpack I have this setup for `.sss`: ` loader: "style-loader!css-loader!postcss-loader?parser=sugarss"`. Compiling with the CLI tool works, but with the loader it doesn't and I get this error:

```
ERROR in ./src/styles/button.sss
Module build failed: Error: Invalid mapping: {"generated":{"line":3,"column":35},"source":"button.sss","original":{},"name":null}
   at SourceMapGenerator_validateMapping [as _validateMapping] (/Volumes/Data/dev/javascript/react/dashboard/node_modules/postcss/node_modules/source-map/lib/source-map-generator.js:271:15)
   at SourceMapGenerator_addMapping [as addMapping] (/Volumes/Data/dev/javascript/react/dashboard/node_modules/postcss/node_modules/source-map/lib/source-map-generator.js:101:14)
   at Stringifier.builder (/Volumes/Data/dev/javascript/react/dashboard/node_modules/postcss/lib/map-generator.js:259:28)
   at Stringifier.decl (/Volumes/Data/dev/javascript/react/dashboard/node_modules/postcss/lib/stringifier.js:56:14)
   at Stringifier.stringify (/Volumes/Data/dev/javascript/react/dashboard/node_modules/postcss/lib/stringifier.js:33:24)
   at Stringifier.body (/Volumes/Data/dev/javascript/react/dashboard/node_modules/postcss/lib/stringifier.js:93:18)
   at Stringifier.block (/Volumes/Data/dev/javascript/react/dashboard/node_modules/postcss/lib/stringifier.js:103:18)
   at Stringifier.rule (/Volumes/Data/dev/javascript/react/dashboard/node_modules/postcss/lib/stringifier.js:60:14)
   at Stringifier.stringify (/Volumes/Data/dev/javascript/react/dashboard/node_modules/postcss/lib/stringifier.js:33:24)
   at Stringifier.body (/Volumes/Data/dev/javascript/react/dashboard/node_modules/postcss/lib/stringifier.js:93:18)
   at Stringifier.root (/Volumes/Data/dev/javascript/react/dashboard/node_modules/postcss/lib/stringifier.js:37:14)
   at Stringifier.stringify (/Volumes/Data/dev/javascript/react/dashboard/node_modules/postcss/lib/stringifier.js:33:24)
   at _class.stringify (/Volumes/Data/dev/javascript/react/dashboard/node_modules/postcss/lib/stringify.js:14:9)
   at _class.generateString (/Volumes/Data/dev/javascript/react/dashboard/node_modules/postcss/lib/map-generator.js:235:14)
   at _class.generateMap (/Volumes/Data/dev/javascript/react/dashboard/node_modules/postcss/lib/map-generator.js:192:14)
   at _class.generate (/Volumes/Data/dev/javascript/react/dashboard/node_modules/postcss/lib/map-generator.js:275:25)
   at LazyResult.stringify (/Volumes/Data/dev/javascript/react/dashboard/node_modules/postcss/lib/lazy-result.js:226:24)
   at /Volumes/Data/dev/javascript/react/dashboard/node_modules/postcss/lib/lazy-result.js:163:27
@ ./src/App.js 19:14-44

2 new messages since 7:12 PM
Files
```

**Comment by ai** [comment]
Sorry. I will have time to see it only on weekend :(.

**Comment by ai** [comment]
Sorry. I will have time to see it only on weekend :(.

**Comment by ai** [comment]
Sorry for big delay.

OI found a problem. It was a SugarSS issue. Fixed in 0.1.3: https://github.com/postcss/sugarss/commit/8eebbcaa0c67974c3794eeaf6bcc6cf30d2f7aa3
